.travis.yml

```diff
@@ -0,0 +1,35 @@
+sudo: false
+language: go
+go:
+- 1.5.4
+- 1.6.2
+env:
+  global:
+    - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/intelsdi-x/snap-plugin-publisher-heka
+    - HEKA_SOURCE=/home/travis/gopath/src/github.com/mozilla-services/heka
+  matrix:
+    - TEST=unit
+# NOTE: addons required for Heka:
+addons:
+  apt:
+    packages:
+      - protobuf-compiler
+      - cmake
+      - libgeoip-dev
+before_install:
+- go get github.com/tools/godep
+- if [ ! -d $SNAP_PLUGIN_SOURCE ]; then mkdir -p $HOME/gopath/src/github.com/intelsdi-x; ln -s $TRAVIS_BUILD_DIR $SNAP_PLUGIN_SOURCE; fi # CI for forks not from intelsdi-x
+- go get github.com/mozilla-services/heka || true
+install:
+- export TMPDIR=$HOME/tmp
+- mkdir -p $TMPDIR
+- cd $HEKA_SOURCE
+- ./build.sh
+- cd $SNAP_PLUGIN_SOURCE # change dir into source
+- export GOPATH=$HEKA_SOURCE/build/heka:$GOPATH
```

Comment:
`build.sh` takes care of setting up the go environment, so setting GOPATH ourselves should not be needed here.